consul/src/main/scala/io/buoyant/consul/v1/BaseApi.scala

```diff
@@ -49,9 +49,15 @@ trait BaseApi extends Closable {
   private[v1] def mkreq(
     method: http.Method,
     path: String,
+    consistency: Option[ConsistencyMode],
     optParams: (String, Option[String])*
   ): http.Request = {
-    val params = optParams.collect { case (k, Some(v)) => (k, v) }
+    val consistencyMode = consistency match {
+      case Some(ConsistencyMode.Consistent) => ("consistent", Some(true.toString))
+      case Some(ConsistencyMode.Stale) => ("stale", Some(true.toString))
+      case _ => ("default", None)
```

Comment:
This is weird because the "default" string is never used, it just gets filtered out.  See above suggestion.